resources/benchmark-runner.mjs

```diff
@@ -36,15 +36,25 @@ class Page {
         });
     }
 
-    querySelector(selector) {
-        const element = this._frame.contentDocument.querySelector(selector);
+    getParent(path) {
```

Comment:
`getParent` => `_getParent`
Since this does not wrap the node it should probably not be used externally.

Reply:
done